posit-dev/positron
# Data Explorer: Add column/row header and data cell context menus for data copying or other actions

### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any GitHub issues that are related.

This PR contains a big push on adding column header, row header, and cell context menus to the Positron Data Explorer. Now that the context menus are in place, we can get feedback on them.

Note: I've renamed the `anchor: HTMLElement` parameter / property to `anchorElement: HTMLElement` because of the addition of a new `anchorPoint` parameter / property.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues.

Note that right clicking on a column header, row header, or cell, will ensure that the column header, row header, or cell is scrolled into view.
